packages/synapse-react-client/src/lib/containers/MarkdownPopover.tsx

```diff
@@ -1,124 +1,105 @@
-import { PositioningStrategy } from '@popperjs/core'
 import React, { useState } from 'react'
-import { Button } from '@mui/material'
-import { Placement } from 'react-bootstrap/esm/Overlay'
-import { ButtonVariant } from 'react-bootstrap/esm/types'
-import { usePopper } from 'react-popper'
-import { CSSTransition } from 'react-transition-group'
+import {
+  Button,
+  Box,
+  TooltipProps,
+  tooltipClasses,
+  ButtonProps,
+} from '@mui/material'
 import MarkdownSynapse, {
   MarkdownSynapseProps,
 } from './markdown/MarkdownSynapse'
 import { Typography } from '@mui/material'
+import LightTooltip from '../components/styled/LightTooltip'
 
 export type MarkdownPopoverProps = {
   children: JSX.Element
   contentProps: MarkdownSynapseProps
-  style?: React.CSSProperties
-  placement?: Placement
+  sx?: TooltipProps['sx']
+  placement?: TooltipProps['placement']
   showCloseButton?: boolean
   actionButton?: {
     content: React.ReactElement
-    variant: ButtonVariant
+    color?: ButtonProps['color']
+    variant?: ButtonProps['variant']
     onClick: () => void
-    closePopoverOnClick: boolean
+    closePopoverOnClick?: boolean
```

Comment:
Make props optional to prevent Typescript warnings in HelpPopover and CertificationQuiz, where the `actionButton` configurations don't include all of these values